src/hive/components/screens/home/core.cljs

```diff
@@ -39,18 +37,18 @@
   destination"
   [props]
   (let [height   (* 80 (count (:user/places props)))]
-    [:> react/View {:style {:height height :paddingTop 100 :paddingLeft 10}}
+    [:> react/View {:height height :paddingTop 100 :paddingLeft 10}
      (for [target (:user/places props)
            :let [distance (/ (geometry/haversine (:user/position props) target)
                              1000)]]
        ^{:key (:id target)}
        [:> react/TouchableOpacity
          {:style {:flex 1 :flexDirection "row"}
           :on-press #(run! work/transact! (choose-route target props))}
-         [:> react/View {:style {:flex 0.2 :alignItems "center" :justifyContent "flex-end"}}
+         [:> react/View {:flex 0.2 :alignItems "center" :justifyContent "flex-end"}
```

Comment:
Is there any way to style the app independently?

Reply:
I don't understand what you mean by that ... Can you give me an example?

Reply:
@carocad I mean using external styles and not defining every detail inside cljs code.

Reply:
@mehdisadeghi absolutedly :)

You could define those as json files and then read them as cljs maps like `(js->clj (js/require "my-json-style.json")`

You can also create js files and add those to the project. Although with those I would be a bit cautious since I think mixing is an alpha feature so it might blow on your face